src/UI/RestBundle/Controller/Security/SecurityController.php

```diff
@@ -84,7 +84,7 @@ public function postLoginAction(ParamFetcher $fetcher): array
      * @RequestParam(name="email", strict=false, default="", description="Unique email")
      * @RequestParam(name="password", strict=false, default="", description="Plain password")
      *
-     * @View(statusCode=202, serializerGroups={"Identifier", "Basic"})
+     * @View(statusCode=201, serializerGroups={"Identifier", "Basic"})
```

Comment:
Why tests are passing now? Its this param ignored?
See https://github.com/jorge07/ddd-playground/blob/master/tests/UI/RestBundle/Controller/User/UserControllerTest.php#L47

Reply:
Yes, I was surprised too! Really I don't know!